File: Beecrowd/1 - Beginner/1101 - 1190/1118 - Several Scores with Validation/C/1118_SeveralScoreswithValidation.c
#include <stdio.h>

int main() {
    double number, average, sum = 0;
    int count = 0, option;

    while (count < 2) {
        scanf("%lf", &number);

        if (number >= 0 && number <= 10) {
            sum += number;
            count++;
        } else printf("nota invalida\n");
    }

    average = sum / 2;

    printf("media = %.2lf\n", average);

    while (1) {
        printf("novo calculo (1-sim 2-nao)\n");
        scanf("%d", &option);

        if (option == 1) {
            sum = 0;
            count = 0;

            while (count < 2) {
                scanf("%lf", &number);

                if (number >= 0 && number <= 10) {
                    sum += number;
                    count++;
                } else printf("nota invalida\n");
            }

            average = sum / 2;

            printf("media = %.2lf\n", average);
        } else if (option == 2) break;
    }

    return 0;
}

File: Beecrowd/1 - Beginner/1101 - 1190/1118 - Several Scores with Validation/1118_SeveralScoreswithValidation.py
number_sum = 0
count = 0

while count < 2:
    number = float(input())
    
    if 0 <= number <= 10:
        number_sum += number
        count += 1
    else:
        print("nota invalida")

average = number_sum / 2

print(f"media = {average:.2f}")

while True:
    print("novo calculo (1-sim 2-nao)")
    option = int(input())
    
    if option == 1:
        number_sum = 0
        count = 0
        
        while count < 2:
            number = float(input())
            
            if 0 <= number <= 10:
                number_sum += number
                count += 1
            else:
                print("nota invalida")
        
        average = number_sum / 2
        
        print(f"media = {average:.2f}")
    elif option == 2:
        break
